pkg/cluster/denyassignment.go

```diff
@@ -6,14 +6,13 @@ package cluster
 import (
 	"context"
 
+	"github.com/Azure/ARO-RP/pkg/env"
 	"github.com/Azure/ARO-RP/pkg/util/arm"
-	"github.com/Azure/ARO-RP/pkg/util/deployment"
 	"github.com/Azure/ARO-RP/pkg/util/stringutils"
 )
 
 func (m *manager) createOrUpdateDenyAssignment(ctx context.Context) error {
-	if m.env.DeploymentMode() != deployment.Production {
-		// only need this upgrade in production, where there are DenyAssignments
+	if m.env.FeatureIsSet(env.DisableDenyAssignments) {
```

Comment:
Its a nit, but I kinda like when variables starts with type `FeatureDisableDenyAssignments` so when doing auto-completes easier to filter. wdyt?

Reply:
I agree.  It makes the serialisation a little bit more fiddly but I'll see if I can find a neat way to do this.